scripts/update_deps.sh

```diff
@@ -1,37 +0,0 @@
-#!/bin/sh
```

Comment:
I wonder if the script changes should be a separate PR to make the backport/cherry-picking process easier.

Reply:
Good point :)

Reply:
Fixed.